src/scheduled-query/azext_scheduled_query/custom.py

```diff
@@ -11,13 +11,17 @@ def create_scheduled_query(client, resource_group_name, rule_name, scopes, condi
     from .vendored_sdks.azure_mgmt_scheduled_query.models import (ScheduledQueryRuleResource,
                                                                   ScheduledQueryRuleCriteria,
                                                                   ConditionFailingPeriods)
+    from knack.util import CLIError
 
     for cond in condition:
         if cond.failing_periods is None:
             cond.failing_periods = ConditionFailingPeriods(
                 min_failing_periods_to_alert=1,
                 number_of_evaluation_periods=1
             )
+        else:
+            if cond.failing_periods.min_failing_periods_to_alert > cond.failing_periods.number_of_evaluation_periods:
```

Comment:
It should be >=

Reply:
Is it ok to have same period and failing period?

Reply:
Yes, this is the default. It means one failure in one last period.

Reply:
So this value should be correct since this condition will raise the CLIError.

Reply:
But why return error on valid values?